app/routes.py

```diff
@@ -1,2 +1,101 @@
-from flask import Blueprint
+from app import db
+from app.models.planet import Planet
+from flask import Blueprint, jsonify, make_response, request, abort
+
+
+planets_bp = Blueprint("planets", __name__, url_prefix="/planets")
+
+@planets_bp.route("/teapot", methods=["GET", "POST"])
+def handle_teapot():
+    return make_response("I'm a teapot!", 418)
+
+def validate_planet(planet_id):
+    try:
+        planet_id = int(planet_id)
+    except:
+        abort(make_response({"message":f"Planet {planet_id} invalid"}, 400))
+
+    planet = Planet.query.get(planet_id)
+
+    if not planet:
+        abort(make_response({"message":f"Planet {planet_id} not found"}, 404))
+
+    return planet
+
+@planets_bp.route("/<planet_id>", methods = ["GET"])
+def read_one_planet(planet_id):
+    planet = validate_planet(planet_id)
+    return {
+        "id": planet.id,
+        "order_from_sun": planet.order_from_sun,
+        "name": planet.name,
+        "description": planet.description,
+        "gravity": planet.gravity
+    }
```

Comment:
in your `Planet` class you can create an instance method to return this information. Then you can use the instance method over and over again.